### DISCUSSION TITLE
https://ecust-leap.github.io/cise/kao-yan/2017/PHIKN1GHT

### DISCUSSION THREAD

**Discussion by giscus** [ECUST-Leap/ecust-leap.github.io#291] [评论区]
# https://ecust-leap.github.io/cise/kao-yan/2017/PHIKN1GHT

衷心祝愿所有同学都不用阅读这篇文章的任何内容。但如果你刚刚遭遇到了考研的失败，或是已经萌生了二战的想法，我只希望你知道，即使在这条路上，你也并不孤独。

https://ecust-leap.github.io/cise/kao-yan/2017/PHIKN1GHT

**Comment by HicoderDR** [comment]
前排沙发
